Foundation/NSOperation.swift

```diff
@@ -47,7 +47,6 @@ open class Operation : NSObject {
 #endif
     }
     
-    /// - Note: Operations that are asynchronous from the execution of the operation queue itself are not supported since there is no KVO to trigger the finish.
     open func start() {
```

Comment:
So the problem now is that if someone overrides `main`  but not `start` and detaches from the current operation's queue.